config/jobs/kubernetes-sigs/node-feature-discovery/node-feature-discovery-presubmits-master.yaml

```
@@ -42,3 +42,25 @@ presubmits:
       - image: golang:1.19
         command:
         - scripts/test-infra/build.sh
+  - name: pull-node-feature-discovery-verify-e2e-test-master
+    branches:
+    - ^main$
+    - ^master$
+    max_concurrency: 1
+    decorate: true
+    annotations:
+      testgrid-dashboards: sig-node-node-feature-discovery
+      testgrid-tab-name: e2e-test-presubmit
+      description: "Run end-to-end tests for node-feature-discovery before merging a pull request"
+      testgrid-alert-email: markus.lehtonen@intel.com,feruzjon.muyassarov@intel.com
+    spec:
+      containers:
+      - image: golang:1.19
+        command:
+        - bash
+        args:
+        - -c
+        - apt update && apt install -y jq && scripts/test-infra/test-e2e-presubmit.sh
```

We probably don't need jq here(?)
```suggestion
        - scripts/test-infra/test-e2e-presubmit.sh
```

---

yep we don't indeed it for the new script indeed